docs/guide/writing/docs-principles.rst

```diff
@@ -1,10 +1,9 @@
-Documentation principles
-========================
+Documentation principles for creating great content
```

Comment:
I don't love "content" here at all. It also kinda redundant if we're already saying Documentation.

How about 

> Principles for creating great documentation